### ISSUE TITLE
扩大遮罩尺寸出现蒙板

### ISSUE THREAD

**Issue by qiuzi** [kleinlee/DH_live#25]
![image](https://github.com/user-attachments/assets/ff6e2384-4fd3-450a-8389-d21195de062d)

修改三个文件：`talkingface/data/few_shot_dataset.py`  `talkingface/utils.py` `talkingface/run_utils.py` 
分别对256 与512 图像尺寸扩大一倍出现上图现象

请问还需要修改哪里使其正常显示？

**Comment by kleinlee** [comment]
可以自己研究下。 DH_live代码的分辨率锁定在256，并未过多考虑更高分辨率的情况。但模型构架和数据处理是支持高分辨率的。

**Comment by qiuzi** [comment]
用gpt要的512模型卷层
`class DINet_five_Ref(nn.Module):
    def __init__(self, source_channel, ref_channel, cuda=True):
        super(DINet_five_Ref, self).__init__()
        self.source_in_conv = nn.Sequential(
            SameBlock2d(source_channel, 32, kernel_size=7, padding=3),
            DownBlock2d(32, 64, kernel_size=3, padding=1),
            DownBlock2d(64, 128, kernel_size=3, padding=1),
            DownBlock2d(128, 128, kernel_size=3, padding=1)  # 新增的下采样层
        )
        self.ref_in_conv = nn.Sequential(
            SameBlock2d(ref_channel, 64, kernel_size=7, padding=3),
            DownBlock2d(64, 128, kernel_size=3, padding=1),
            DownBlock2d(128, 256, kernel_size=3, padding=1),
            DownBlock2d(256, 256, kernel_size=3, padding=1)  # 新增的下采样层
        )
        self.trans_conv = nn.Sequential(
            SameBlock2d(384, 128, kernel_size=3, padding=1),
            SameBlock2d(128, 128, kernel_size=7, padding=3),
            DownBlock2d(128, 128, kernel_size=3, padding=1),
            SameBlock2d(128, 128, kernel_size=7, padding=3),
            DownBlock2d(128, 128, kernel_size=3, padding=1),
            SameBlock2d(128, 128, kernel_size=3, padding=1),
            DownBlock2d(128, 128, kernel_size=3, padding=1),
            SameBlock2d(128, 128, kernel_size=3, padding=1),
            DownBlock2d(128, 128, kernel_size=3, padding=1),
            SameBlock2d(128, 128, kernel_size=3, padding=1),
            DownBlock2d(128, 128, kernel_size=3, padding=1),  # 新增的下采样层
        )

        # 其他层保持不变...

        self.out_conv = nn.Sequential(
            SameBlock2d(384, 128, kernel_size=3, padding=1),
            UpBlock2d(128, 128, kernel_size=3, padding=1),
            ResBlock2d(128, 128, 3, 1),
            UpBlock2d(128, 128, kernel_size=3, padding=1),
            UpBlock2d(128, 128, kernel_size=3, padding=1),  # 新增的上采样层
            nn.Conv2d(128, 3, kernel_size=3, padding=1),
            nn.Sigmoid()
        )

    # 其他方法保持不变...

# 使用示例
source_img = torch.zeros([1, 3, 512, 512]).cuda()
source_prompt = torch.zeros([1, 3, 512, 512]).cuda()
ref_img = torch.zeros([1, 30, 512, 512]).cuda()

model_Generator = DINet_five_Ref(6, 30).cuda()
model_Generator.ref_input(ref_img)
out = model_Generator.interface(source_img, source_prompt)`
训练没有报错，推理也正常，正确吗？

**Comment by kleinlee** [comment]
模型结构应该没问题。数据只要通过了train_input_validation_render_model.py验证，那就都没问题

**Comment by galigaligo** [comment]
兄弟，怎么样？解决了吗

**Comment by qiuzi** [comment]
> 兄弟，怎么样？解决了吗

没有解决，最终放弃了
